airflow/www/views.py

```diff
@@ -3084,6 +3084,7 @@ class GraphForm(DateTimeWithNumRunsWithDagRunsForm):
             t.task_id: {
                 "dag_id": t.dag_id,
                 "task_type": t.task_type,
```

Comment:
Do we still need this? If the only use case if to check SubDagOperator, we can use operator_name as well.

Reply:
Yes, I think we still need this since the below code compares the operator name
https://github.com/apache/airflow/blob/8891c6944671d8965a63ddbd59b5c23597e86f0a/airflow/www/static/js/graph.js#L211
and based on that displaying the subdag button -
https://github.com/apache/airflow/blob/8891c6944671d8965a63ddbd59b5c23597e86f0a/airflow/www/static/js/callModal.js#L185-L190

Reply:
Wouldn't it be better to pass `is_subdag` from the webserver instead?

Reply:
Yes, we can. I'll update the code. :)

Reply:
@bbovenzi Would it make sense to raise a separate PR for this change? because I think it's a separate task.